apps/web/controllers/pets/new.rb

```diff
@@ -0,0 +1,13 @@
+module Web::Controllers::Pets
+  class New
```

Comment:
It is really cool that Lotus enforces on action per class, I'm trying to do this on rails, and I like it!